src/AccessibilityInsights.SharedUx/Controls/CustomControls/ScannerResultCustomListControl.xaml.cs

```diff
@@ -19,6 +21,7 @@ namespace AccessibilityInsights.SharedUx.Controls.CustomControls
     public partial class ScannerResultCustomListControl : UserControl
     {
         private ScannerResultCustomListContext _controlContext;
+        private bool _ignoreNextSelectionChange;
 
         public IEnumerable ItemsSource
```

Comment:
Are these still needed? We want people to use the setItemSource method instead of setting the items directly